daemon/status.go

```diff
@@ -141,7 +142,7 @@ func (h *getHealthz) getStatus(d *Daemon) models.StatusResponse {
 		sr.Cilium = &models.Status{State: models.StatusStateOk, Msg: "OK"}
 	}
 
-	if d.DebugEnabled() {
+	if config.DebugEnabled() {
```

Comment:
I fail to understand this. While I was reading the PR I thought `agentConfig` was the runtime configuration and `config` was the list of constants available for configuration. Why is `config` checking if Debug is enabled? should this method belong to `agentConfig`?

Reply:
We had accessor functions for this and I kept them. We can remove them in favour of accessing `agentConfig` directly. I will change this.